### PR TITLE
docs: section on insufficient l2callvaue -> retryable failure 

### PR DESCRIPTION
**Pull request by DZGoldman**

### CODE REVIEW

File: docs/L1_L2_Messages.md [hunk 1]
```diff
@@ -70,12 +70,15 @@ Intuitively: if a user does not desire immediate redemption, they should provide
   `Calldata:` data encoding the L2 contract call.
   `Calldata Size:` CallData size.
 
-### Important Note About Base Submission Fee
+###  Ticket Creation Failure: Base Submission Fee (IMPORTANT)
```
Comment (gzeoneth) [inline docs/L1_L2_Messages.md:73]:
```suggestion
### Ticket Creation Failure: Base Submission Fee (IMPORTANT)
```

File: docs/L1_L2_Messages.md [hunk 2]
```diff
@@ -70,12 +70,15 @@ Intuitively: if a user does not desire immediate redemption, they should provide
...
 
 If an L1 transaction underpays for a retryable ticket's base submission free, the retryable ticket creation on L2 simply fails. Given that this potentially breaks the atomicity of the L1 / L2 transactions, applications should avoid this scenario. The current base submission fee returned by `ArbRetryableTx.getSubmissionPrice` increases once every 24 hour period by at most 50% of its current value. Since any amount overpaid will be credited to the `credit-back-address`, it is highly recommended that applications judiciously overpay relative to the current price.
 
 In a future release, the base submission fee will be calculated using the 1559 `BASE_FEE` and collected directly at L1; underpayment will simply result in the L1 transaction reverting, thus avoiding the complications above entirely.
 
+###  Ticket Creation Failure: L2 Callvalue
```
Comment (gzeoneth) [inline docs/L1_L2_Messages.md:79]:
```suggestion
### Ticket Creation Failure: L2 Callvalue
```

File: docs/L1_L2_Messages.md [hunk 3]
```diff
@@ -70,12 +70,15 @@ Intuitively: if a user does not desire immediate redemption, they should provide
...
+Likewise, a retryable ticket will fail to be created if insufficient callvalue is provided to L2 for execution. Note that this value can be supplied from the L1 transaction (recommended) or, if not supplied from L1, will be deducted from the sender's L2 account.
```
Comment (gzeoneth) [inline docs/L1_L2_Messages.md:80]: how about
> will be deducted from the sender's L2 account **(or its L2 alias if it is a contract)**

**Comment by DZGoldman** [comment]
@gzeoneth updated (sry this had slipped through)

**Comment by joshuacolvin0** [comment]
docs moved to Nitro repo

**Pull request closed without merging**